src/index.html

```diff
@@ -10,8 +10,164 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="body">
+    <form
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/   create-application"
+      class="form">
```

Comment:
the closing tag should be on the same level as opening tag
```suggestion
    <form
      method="post"
      action="https://mate-academy-form-lesson.herokuapp.com/   create-application"
      class="form"
    >
```